src/modules/overlay-renderer/overlay-renderer.ts

```diff
@@ -66,8 +66,8 @@ export class OverlayRenderer implements Renderer {
         position: absolute;
         transform-origin: 0px 0px;
       `),
-      interactive: this.stylesheet.addStylesheet(`pointer-events: all`),
-      nonInteractive: this.stylesheet.addStylesheet(`pointer-events: none`),
+      interactive: this.stylesheet.addStylesheet(`pointer-events: none`),
+      nonInteractive: this.stylesheet.addStylesheet(`pointer-events: all`),
```

Comment:
@stephenwf I noticed these should be changed, but wasn't sure if there was a good way to test this and the 'atlas-static-image' class below too

Reply:
Hmm, are you sure? The interactive property was intended to indicate if the HTML in the overlay is interactive (e.g. if there was text you could select it, or a button you could click it). If it's marked as `nonInteractive` then it the _atlas_ events such as `onClick` would still work.

Reply:
no, not sure at all, hence the question.  Will swap it back

Reply:
reverted